innovation.js

```diff
@@ -1806,61 +1823,80 @@ function (dojo, declare) {
                 var top_card = pile[pile.length - 1];
                 var top_card_id = self.getCardIdFromHTMLId(top_card.id);
                 var top_card = self.cards[top_card_id];
-                HTML_action += dojo.string.substitute("<p>" + _("If you do, it will cover ${age} ${card_name} and your new ressource counts will be:") + "<p>",
-                    {
-                        'age': self.square('N', 'age', top_card.age, 'in_log'),
-                        'card_name': "<span class='card_name'>" + _(top_card.name) + "</span>"
-                    });
-            }
-            else {
-                HTML_action += "<p>" + _("If you do, your new ressource counts will be:") + "</p>";
+                if (self.cities_expansion_enabled || self.echoes_expansion_enabled) {
+                    HTML_action += dojo.string.substitute("<p>" + _("If you do, it will cover ${age} ${card_name}, you will have a total score of ${score}, and your new featured icon counts will be:") + "<p>",
+                        {
+                            'age': self.square('N', 'age', top_card.age, 'in_log'),
+                            'card_name': "<span class='card_name'>" + _(top_card.name) + "</span>",
+                            'score' : new_score
+                        });
+                } else {
+                    HTML_action += dojo.string.substitute("<p>" + _("If you do, it will cover ${age} ${card_name} and your new ressource counts will be:") + "<p>",
+                        {
+                            'age': self.square('N', 'age', top_card.age, 'in_log'),
+                            'card_name': "<span class='card_name'>" + _(top_card.name) + "</span>"
+                        });
+                }
+            } else {
+                if (self.cities_expansion_enabled || self.echoes_expansion_enabled) {
+                    HTML_action += "<p>" + dojo.string.substitute(_("If you do, you will have a total score of ${score} and your new featured icon counts will be:"), {'score' : new_score}) + "</p>";
+                } else {
+                    HTML_action += "<p>" + _("If you do, your new ressource counts will be:") + "</p>";
```

Comment:
spell check here

Reply:
That's an existing string that is already translated so I couldn't fix the typo without triggering retranslation.